torchvision/datasets/utils.py

```diff
@@ -197,6 +197,10 @@ def _save_response_content(response, destination, chunk_size=32768):
         pbar.close()
 
 
+def _is_tarxz(filename):
+    return filename.endswith(".tar.xz")
```

Comment:
If this doesn't work on Python 2, can we maybe skip support for this format for Python 2?

Reply:
I agree, updated the [`extract_archive`](https://github.com/pytorch/vision/pull/1361/files#diff-c3782ceaa9828c1648c702a426d6a808R231) function to extract `.tar.xz` only under Python3 (`PY3`)

I purposely left `_is_tarxz(filename)` untouched because the function itself is working regardless the platform.